common/lib/protocol-definitions/src/consensus.ts

```diff
@@ -98,3 +99,36 @@ export interface IProtocolState {
 export interface IProcessMessageResult {
     immediateNoOp?: boolean;
 }
+
+/**
+ * Audience represents all clients connected to the op stream, both read-only and read/write.
+ */
+ export interface IAudience extends EventEmitter {
```

Comment:
`protocol-base` needs it, so that would imply adding a dependency to `container-definitions`, which is under `packages/common`, not `common/lib`. Haven't tried it yet, but I don't think that dependency is viable.

Reply:
No longer an issue https://github.com/microsoft/FluidFramework/pull/10818#discussion_r910409346